### DISCUSSION TITLE
真的很好用,感谢!!!

### DISCUSSION THREAD

**Discussion by puyoaiyo** [ag2s20150909/TTS#74] [General]
真的很好用,感谢!!!

**Comment by gmzim** [comment]
现在用不成了，期待更新。

**Comment by testwind** [comment]
@gmzim 
更新的一点小修复，可以继续使用了

**Reply by jiefly-666** [reply]
大佬你好 可是我下载了2025版的 进去没刷新播音员列表 之后就闪退进不去了啊

**Comment by DBSDBS** [comment]
GREAT app.... THANK you.

Is there a way APK to be fully in English?
If you tell me which files have to be translated, I'll use ChatGPT to translate them.

IF you have no time, just tell me how to make .APK file after translations, and I'll do and post here.

Thank you in advance.
